### PR TITLE
Add draft of DSAA wrapped blog post

### PR DESCRIPTION
**Pull request by chloepouprom**
# Summary

Provide a summary of your blog post here.

# Checklist (Writer)

- [x] I can successfully knit the blog post Rmd file.

- [x] Figures, graphs, tables render properly in the blog post Rmd file.

- [x] Abbreviations are expanded/explained.

- [x] The blog post Rmd file header contains the post title, post description, author information, and categories.

- [x] The blog post content (Rmd file, figures, tables, etc.) does not contain any PHI.

# Checklist (AA Blog Team Reviewers)

- [ ] I have updated the blog's draft status from 'TRUE' to 'FALSE'

- [ ] I have updated the date on the blog post to reflect the release date

- [ ] I have renamed the blog post directory so the date matches the release date

- [ ] I have re-rendered the blog post and all required files are present

### CODE REVIEW

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+![](dsaa_wrapped.PNG)
+
+Enjoy some stats!!
+
+# Projects
+
+2022 was a busy year with lots of projects!!
+
+- Total \# of projects deployed: 7
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:31]: Is this number correct? I feel like there are more...
ED wait times, SMH IRT staffing optimization, chartwatch surgical, ops centre unit level tiles, hemodialysis, ICH

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    self_contained: false
+draft: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+The end of the year is a time for reflection, reminiscing, and sharing screenshots of our Spotify Wrapped. For those who are unfamiliar, Spotify Wrapped is a yearly marketing campaign by Spotify, where they compile streaming data and create a personalized graphic for each user. ^[We surveyed the members on DSAA and found that the most listened music genre is: ???]
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:19]: Could be fun to ask people what their top genre on Spotify Wrapped (or the Apple Music equivalent) was and we could include this here :D

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+# Projects
+
+2022 was a busy year with lots of projects!!
+
+- Total \# of projects deployed: 7
+
+- Most viewed project: Ops Centre
+
+    - The Ops Centre project is....
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:35]: Todo

Comment (chloepouprom) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:35]: Also, I don't have exact view counts, but I'm assuming Ops Centre is our most used/viewed!

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    
+- Project with the most team members whose names start with _M_^: Hemodialysis
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:37]: Meggie, Maitreyee, Mitch

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+    - The Hemodialysis project is...
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:39]: Todo

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+# Product families
+
+2022 was the year of _product families_.
+
+
+![](https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fimages.squarespace-cdn.com%2Fcontent%2Fv1%2F58b3cae5d1758e0132a008c6%2F1582078446179-VU0VAYNPOV4P3JIBKNIY%2FIt%2527s%2BAbout%2BFamily%2BLogo%2B1500.png%3Fformat%3D1000w)
+
+What's a product family?
+
+> A product family is....
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:51]: Do we have a definition for "product families" somewhere?

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+
+Some product family stats!
+
+- Number of product families: ??
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:56]: I think this is... 6? Or 5?

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+# DSAA and improvement
+
+2022 was a year where we focused on improving existing operations.
+
+- \# of sprint improvement weeks: 3
+
+    - The sprint improvement week is a week where we....
+    
+    - Some notable achievements from past sprint improvement weeks are ...
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:68]: Todo

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    
+- \# of blog posts: 5
+  
+    - 2022 is when we re-launched this blog!
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:72]: Do we have any stats on views/traffic/engagement? We could add something here!

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+    
+- \# of journal club sessions lead by the Advanced Analytics team: at least 16
+
+    - We covered the following topics: [add links to ethics & DEI], anomaly detection^[blog post coming soon! :-)], survival analysis, Canadian healthcare.
+    
+- \# of "sheets" created: 3
+
+- \# of Advanced Analytics mini-demos: 8
+
+    - Mini-demos are lead by a different Advanced Analytics team member..... Add link to Derek's funneljoin here
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:82]: Todo

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 1 above)
Comment (derekbeaton) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:31]: ID consult, CHARTWatch 2.0.

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 4 above)
Comment (derekbeaton) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:37]: alternate: product family with most Ms?

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 6 above)
Comment (derekbeaton) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:51]: Cathy will write a blog post on it. I think she'd be happy to give us a sentence or two for this

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 7 above)
Comment (derekbeaton) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:56]: 6, but also like 6.5. Data Fulfillment is the elusive DIG-only family, and we have special projects that don't fit into any families

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 1 above)
Comment (derekbeaton) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:31]: so maybe let's say at least 8-9 within the year. I do think there were some data fulfillment ones, too (e.g., a PACS one).

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 2 above)
Comment (meggie-d) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:19]: looks like the top answer was pop (and some interesting variations of indie ~ indie soul, indie pop, indie Québécois)

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 5 above)
Comment (meggie-d) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:39]:
```suggestion
    - The Hemodialysis project is a weekly email alert that gets sent to clinicians at the St. Michael's Hospital hemodialysis clinics. The goal of the project is to predict a patient's risk of a hospital visit in the 30 days following their dialysis sessions.
```

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+What's a product family?
+
+> A product family is....
+
+
+Some product family stats!
+
+- Number of product families: ??
+
+- Product family with the most projects: Patient Flow??
```
Comment (meggie-d) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:58]: did a rough count of the sub-folders in each of the PF clickup spaces - this is what i found based on old/new projects combined!
- patient flow: 8
- ai triage: 10
- medical imaging: 2
- workforce optimization: 11
- decision guidance: 9
- data fulfillment: 3

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+
+# DSAA and improvement
+
+2022 was a year where we focused on improving existing operations.
+
+- \# of sprint improvement weeks: 3
+
+    - The sprint improvement week is a week where we....
```
Comment (meggie-d) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:66]:
```suggestion
    - The sprint improvement week is one full week dedicated to DSAA process improvement work in the areas of data infrastructure, source system optimization, DevOps, data governance, databases, analytics and project management processes, and more! The DSAA team works together to clear a backlog of tasks, and then we share our accomplishments through demos at the end of the week.
```

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+    
+    - Some notable achievements from past sprint improvement weeks are ...
+    
+- \# of blog posts: 5
+  
+    - 2022 is when we re-launched this blog! 
+    
+- \# of journal club sessions lead by the Advanced Analytics team: at least 16
+
+    - We covered the following topics: [add links to ethics & DEI], anomaly detection^[blog post coming soon! :-)], survival analysis, Canadian healthcare.
```
Comment (meggie-d) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:76]:
```suggestion
    - We covered the following topics: [AI ethics, equity, & inclusion](https://github.com/LKS-CHART/AA_journal_club/tree/main/2022_01_Ethics_Bias_Inclusion), anomaly detection^[blog post coming soon! :-)], survival analysis, Canadian healthcare.
```

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+    
+    - Speaking of pizza....
+    
+- \# of pizzas made by DSAA team members: $\infty$
+
+[add pics of pizza here???]
+
+- Most requested album: The Charlie Brown Christmas soundtrack
+
+    - [something baout solo work together channel]
```
Comment (meggie-d) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:104]:
```suggestion
- Most requested album: The Charlie Brown Christmas soundtrack

    - In an effort to maintain some of the pre-pandemic joy of working independently but with others around, we created a slack channel called #solo-work-together, where we can hop on a call and work with others while listening to some background music, which ranges from lo-fi beats to Charlie Brown!
```

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd (hunk 1 above)
Comment (derekbeaton) [reply _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:31]: We have 9 big ones **from July through December**: hemo, ID consult, CW2.0, Unit Level Tiles, SMH IRT, CW Surg, Ops Centre alerting, ED wait times, ICH

File: _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+- **Product family** (roughly comparable to a product line) is a group of related products that are developed and maintained by a single cross-functional team at DSAA. 
+
+- We adopted this custom framework in order to allow staff to work effectively within persistent teams and to leverage resources across products the best we can. The products under each product family are related by common product value and/or shared domain expertise requirement.
+
+
+Some product family stats!
+
+- Number of product families: 6
+
+- Product family with the most projects: The _Patient Flow_ product family ^[Fun fact: The _Patient Flow_ product family is also the product family with the most Fast & Furious memes.]
```
Comment (chloepouprom) [inline _posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd:61]: Cathy confirmed that there are 6 families and that Patient Flow has the most projects

**Pull request merged**